koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -55,15 +55,18 @@ describe("About Applying What We Have Learnt", function() {
       }
     }
 
-    expect(sum).toBe(FILL_ME_IN);
+    expect(sum).toBe(233168);
   });
 
   it("should add all the natural numbers below 1000 that are multiples of 3 or 5 (functional)", function () {
 
-    var sum = FILL_ME_IN;    /* try chaining range() and reduce() */
 
-    expect(233168).toBe(FILL_ME_IN);
-  });
+    var sum = _.range(1, 1000)
+      .filter(function(n) {return n % 3 === 0 || n % 5 === 0;})
```

Comment:
:+1: I like the idea of using filter before reduce. This makes it quite a bit cleaner than just using `reduce` with `if`